secure-api-gateway-ob-uk-rs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rs/server/common/util/link/LinksHelper.java

```diff
@@ -27,7 +27,7 @@ public class LinksHelper {
     private static final String PISP = "pisp";
     private static final String RS = "rs";
     private static final String OPEN_BANKING = "open-banking";
-    private static final String DOMESTIC_PAYMENTS_CONSENT = "domestic-payments-consent";
+    private static final String DOMESTIC_PAYMENTS_CONSENT = "domestic-payment-consents";
```

Comment:
Update the property name to `DOMESTIC_PAYMENT_CONSENTS` to match with the value

Reply:
Fixed